resources/ext/cli/ruby.erb

```diff
@@ -1,6 +1,18 @@
 #!/usr/bin/env bash
 
+cli_defaults=${INSTALL_DIR}/cli/cli-defaults.sh
+
+CLASSPATH=${INSTALL_DIR}/<%= EZBake::Config[:uberjar_name] %>
+
+if [ -e "$cli_defaults" ]; then
+  . $cli_defaults
+  if [ $? -ne 0 ]; then
+    echo "Unable to initialize cli defaults, failing irb subcommand." 1>&2
+    exit 1
+  fi
+fi
+
```

Comment:
We might be able to reduce the duplication a little bit.  Some of the duplication, though, seems desirable to me.  At least the bits about setting a default value for the CLASSPATH variable and checking if the cli_defaults file exists before trying to source it would seem good to duplicate between the files for error handling / robustness within the individual scripts.  My preference would be to leave this in for now but would be curious to hear if others, like @jpinsonault have any strong opinions about this.